services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/operations/OperationType.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.enhanced.dynamodb.internal.operations;
+
+import software.amazon.awssdk.annotations.SdkInternalApi;
+
+@SdkInternalApi
+public enum OperationType {
+    ANY("Any"), //unknown value
```

Comment:
Should it be `UNKNOWN_TO_SDK_VERSION(null);` to be consistent with the service enum?

Reply:
Sounds better, lm check that out.

Reply:
Maybe just UNKNOWN? Because this is internal, why would we even need an UNKNOWN value? Can't we just always specify it for our operations?

Reply:
We don't always know the operation in all code paths flowing through an operation. Especially on the read path, we make transforms and use extensions without coming directly from an operation. An example is the `readAndTransform..Item` operations in `EnhancedClientUtils`. 
We could use null instead, which I don't prefer but I don't feel strongly about it.